api-schema/src/main/java/org/stellar/anchor/api/sep/SepTransactionStatus.java

```diff
@@ -47,4 +49,13 @@ public String getName() {
   public String getDescription() {
     return description;
   }
+
+  public static boolean isValid(String status) {
+    return Arrays.stream(SepTransactionStatus.values())
+        .anyMatch(e -> e.name.equalsIgnoreCase(status));
+  }
+
+  public static void main(String[] args) {
```

Comment:
Looks like tmp debug method, can be removed I guess?

Reply:
Removed